website/docs/getting-started/for-engineers.md

```diff
@@ -239,8 +239,14 @@ Import design tokens as CSS variables by adding one of the following lines to th
 // for product applications (Ember apps)
 @import "@hashicorp/design-system-tokens/dist/products/css/tokens.css";
 
+// for Cloud UI email templating
+@import "@hashicorp/design-system-tokens/dist/cloud-email/tokens.css";
+
 // for HashiCorp developer platform
 @import "~@hashicorp/design-system-tokens/dist/devdot/css/tokens.css";
+
+// for HashiCorp web/marketing websites
+@import '@hashicorp/design-system-tokens/dist/marketing/css/tokens.css';
```

Comment:
we have the Webpack module notation (`~@hashicorp`) for the developer platform. I wonder if that's something we should do for the web/marketing website 🤷

Reply:
I [checked](https://github.com/hashicorp/web/blob/67078524f19bf6798f30304957b611de04ffd156/packages/global-styles/src/style.css#L14-L25), and they don't use the `~` that's why I left it out in the documentation.